### ISSUE TITLE
[EPIC] Add CI

### ISSUE THREAD

**Issue by ftisiot** [aiven/metadata-parser#1]
# What is currently missing?

The current repo is a bare minimum without any CI process

- [ ] [Build plumbing for the CI](https://github.com/aiven/metadata-parser/issues/17)
- [ ] [Add secrets and projects information to CI](https://github.com/aiven/metadata-parser/issues/19)

# Is this a feature you would work on yourself?

Please assign yourself the above issue(s) and contribute to adding a CI for the metadata-parser project.

**Comment by inifares23lab** [comment]
Hi, can I work on this?

**Comment by ftisiot** [comment]
Sure! Please go on!On 24 Jun 2023, at 17:51, inifares23lab ***@***.***> wrote:﻿
Hi, can I work on this?

**Comment by inifares23lab** [comment]
Hey
I opened PR #53 to fix issue #52
changing 'networkx=2.8.8' to 'networkx==2.8.8'

I am asking this here because I am declaring the specific python version in the CI.

I realized that in the repo it is specified that python 3.7 is needed, however with version 3.7 pip throws this error:
`ERROR: No matching distribution found for networkx==2.8.8`

- does it mean that the dependency version should be 'networkx<=2.8.8'?
- or that the python version should be higher than 3.7?
